app/src/main/java/org/literacyapp/appstore/dao/converter/ApplicationStatusConverter.java

```diff
@@ -9,19 +9,19 @@
 
     @Override
     public ApplicationStatus convertToEntityProperty(String databaseValue) {
-        Log.i(getClass().getName(), "convertToEntityProperty");
```

Comment:
@sladomic I changed from INFO to DEGBUG since the stuff happening in this class is not so important to keep in logs. I'm thinking we could save all log output at level INFO/WARNING/ERROR in text files for later debugging, while ignoring DEBUG or lower.